### ISSUE TITLE
reuke where is your cheat ?

### ISSUE THREAD

**Issue by Koksupl1234** [reuke/surviv.io-cheat#28]
/\

**Comment by muWander** [comment]
CHEAT! PLEASE, LIVE!!1!11!!!11!1!АДЫН!111АДЫНАДЫН!

**Comment by muWander** [comment]
![image](https://user-images.githubusercontent.com/39914744/42201810-fadd29a8-7ea1-11e8-91ae-7f50303cf499.png)

**Comment by gaylolxddxddxxd** [comment]
fuck if they add anti cheat imma kms lollolol

**Comment by IceHacks** [comment]
just install it in developer mode unpacked.

**Comment by humphreygaming** [comment]
wdym

**Comment by 1tarko** [comment]
@IceHacks How? were the variables changed ?

**Comment by Kalaborative** [comment]
@1tarko yes. the variables changed

**Comment by IceHacks** [comment]
@humphreygaming @1tarko Download the ZIP extract turn on Chrome Developer Mode click "Load unpacked" and select the extracted directory. I have not tested this hack.

**Comment by sebastians17** [comment]
@IceHacks not working right now

**Comment by sebastians17** [comment]
@IceHacks not working

**Comment by sebastians17** [comment]
@IceHacks this hack is not working

**Comment by IceHacks** [comment]
@sebastians17 It's not my hack and I don't know if it works but that is how you install it.

**Comment by Kalaborative** [comment]
Yeah. Pretty sure we definitely have the ID's right. They changed some variables though. If I could work with an older version of the site I might be able to delve into what's changed, but with a game that's over 30k lines of code, that's not going to be easy.

**Comment by TotomInc** [comment]
Doing @IceHacks method works (load unpacked extension from chrome://extensions), the cheat seems to be injected however there is an anticheat from surviv.io which make the page refresh everytime (like every 3 seconds), making the game impossible to play.

**Comment by humphreygaming** [comment]
@TotomInc you can still play tho, if you click the solo/duo/squad button really fast. You bypass the reloading but the cheat does not work. It only shows a blank FPS and LAT. I dont know why it is taking so long

**Comment by neeksor** [comment]
The reload code is in the extension. It is attempting to find the module names to hook in order for the cheat to work. The problem is after the module detection code was added, it appears the surviio developers renamed the exported functions that the module looks for.

**Comment by publiwebhnventas** [comment]
@neeksor as I said, they have discovered this repository and they're constantly looking for changes, maybe "one of us" is part of their team, @reuke please make a private repository (bitbucket or github paid) and fix the extension please

**Comment by publiwebhnventas** [comment]
@humphreygaming nice job forking @reuke repo, renaming stuff and changing the logo 👎

**Comment by publiwebhnventas** [comment]
@humphreygaming at least that file has the hack fixed. bypass adf.ly link: http://www.mediafire.com/file/tjl7hsj08tw520x/surviv.io+cheat.7z

**Comment by Koksupl1234** [comment]
how to use this ?

**Comment by publiwebhnventas** [comment]
@Koksupl1234 

1. download the file and extract on desktop
2. goto google chrome 
3. settings (in chrome)
4. extensions (in chrome)
5. activate developer mode (in chrome)
6. press load unpacked extension button (in chrome)
7. locate the extracted folder
8. done, goto surviv.io and play

**Comment by Koksupl1234** [comment]
Nice is working thank you very much :D @publiwebhnventas

**Comment by Koksupl1234** [comment]
@publiwebhnventas cheat doesnt work again :/

**Comment by humphreygaming** [comment]
use this https://github.com/w3x731/survivIoAim/ @Koksupl1234
